src/ddmd/aliasthis.d

```diff
@@ -49,6 +49,20 @@ extern (C++) final class AliasThis : Dsymbol
 
     override void semantic(Scope* sc)
     {
+        if (semanticRun < PASSsemanticdone)
```

Comment:
Should this be `>=` instead?

BTW: I tried making semanticRun more consistent in this branch, too: https://github.com/rainers/dmd/tree/tg_frontend